artifactory/cli.go

```diff
@@ -518,6 +518,17 @@ func GetCommands() []cli.Command {
 				goRecursivePublishCmd(c)
 			},
 		},
+		{
+			Name:         "pip-config",
```

Comment:
Let's add to this command the alias **pipc** (se other commands with aliases).
The reason why we don't have the alias **goc** for the **go-config** command, us because we didn't wanted people to get confused with the existing **goc** CLI.